src/drive/web/modules/drive/Toolbar/components/UploadButtonItem.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import classNames from 'classnames'
+import UploadButton from 'drive/web/modules/upload/UploadButton'
+import { translate } from 'cozy-ui/transpiled/react'
+import styles from 'drive/styles/toolbar.styl'
+import { compose } from 'redux'
+
+import toolbarContainer from '../toolbar'
+
+const UploadButtonItem = ({ t, displayedFolder, isDisabled }) => (
+  <UploadButton
+    disabled={isDisabled}
+    displayedFolder={displayedFolder}
+    label={t('toolbar.item_upload')}
+    className={classNames(styles['c-btn'], 'u-hide--mob')}
+  />
+)
+
+export default compose(
+  toolbarContainer,
+  translate()
```

Comment:
Nit: we could use useI18n at some point. This can be done via a codemod.

Reply:
Yep 👍 I will do this in a separate PR to avoid bloating this one, it's already big 😅